clientv3/config.go

```diff
@@ -25,7 +25,8 @@ import (
 )
 
 type Config struct {
-	// Endpoints is a list of URLs
+	// Endpoints is a list of URLs.
+	// Make sure to strip scheme:// prefixes since grpc dials by host.
```

Comment:
if the scheme is stripped then how can grpc distinguish between mixed http and https endpoints?